test/Pass/alive0.ll

```diff
@@ -26,4 +26,4 @@ define i32 @alive0_1(i16) local_unnamed_addr #0 {
   ;CHECK-NOT: %X = sext i16 %0 to i32
   ;CHECK-NOT: %r = select i1 %c, i32 %X, i32 -1
   ;CHECK: ret i32 -1
-}
+}
```

Comment:
github says you're missing newlines at the end of this file and others

Reply:
Fixed